internal/controller/compute/statefulserverset/datavolume_controller.go

```diff
@@ -220,9 +220,9 @@ func (k *kubeDataVolumeController) Update(ctx context.Context, cr *v1alpha1.Stat
 }
 
 // isVolumeUpToDate - checks if the lan is up-to-date and update the kube lan object if needed
```

Comment:
One more mixup in the doc :)
```suggestion
// isVolumeUpToDate - checks if the volume is up-to-date and update the kube volume object if needed
```